crates/protos/src/lib.rs

```diff
@@ -0,0 +1,17 @@
+pub mod message;
+
+pub fn add(left: usize, right: usize) -> usize {
```

Comment:
This function whether is called ?

Reply:
This is auto-generated by cargo when calling `cargo new --lib`

Reply:
If that's not needed, maybe consider removing it.